actions/cloud.tsx

```diff
@@ -0,0 +1,99 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Stripe} from '@stripe/stripe-js';
+import {getCode} from 'country-list';
+
+import {Client4} from 'mattermost-redux/client';
+import {Product} from 'mattermost-redux/types/cloud';
+
+import {getConfirmCardSetup} from 'components/payment_form/stripe';
+
+import {StripeSetupIntent, BillingDetails} from 'types/cloud/sku';
+
+export function getProductPrice() {
+    return async () => {
+        let cloudProducts = [] as Product[];
+
+        try {
+            cloudProducts = await Client4.getCloudProducts();
+        } catch (error) {
+            // eslint-disable-next-line no-console
+            console.error(`Error fetching cloud products: ${error}`);
```

Comment:
These were actually debugging ones - I don't think these should be shared to the console/user since its around payments. You're right, they'll be put on the redux error stack.